examples/va/nvidia-mdev/edpm-post-driver/nodeset/values.yaml

```diff
@@ -0,0 +1,148 @@
+# yamllint disable rule:line-length
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-post-driver-nodeset-values
```

Comment:
This needs to be `edpm-nodeset-values` since the lib kustomization is expecting this file [1]

[1] https://github.com/openstack-k8s-operators/architecture/blob/main/lib/dataplane/nodeset/kustomization.yaml#L22

Reply:
normlay each deployment should have its own name without that we have to delete them in between which is not how this is intended to be used.

Reply:
oh never mind this is not the deployment name its the config map name ya we can use the same config map for the replacements and just updated it